src/main/resources/global-build-tools/checkstyle.xml

```diff
@@ -193,8 +196,10 @@
     </module>
     <module name="MissingJavadocType"/>
     <module name="NonEmptyAtclauseDescription"/>
-    <!-- Allow suppression of checkstyle warnings -->
+    <!-- Allow suppression of checkstyle warnings via comments -->
     <module name="SuppressionCommentFilter"/>
```

Comment:
I think `SuppressCommentFilter` must be moved also into the TreeWalker module  - see \
https://checkstyle.sourceforge.io/filters/suppressioncommentfilter.html